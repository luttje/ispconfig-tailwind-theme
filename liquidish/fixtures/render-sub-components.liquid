<div>
    {% render "render-sub-components-sub.liquid" %}
    {% if show %}
        <p>It's show time!</p>
    {% endif %}
</div>
