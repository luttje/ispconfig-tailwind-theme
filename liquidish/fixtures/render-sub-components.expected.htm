<div>
    <div class="flex flex-col gap-4"
        data-foo="bar"
        data-bar="baz"
        wire:ignore
        @click="console.log('clicked')">
        <h1 class="text-2xl">{tmpl_var name="SLOT_VARIABLE"}</h1>
    </div>
    {tmpl_if name="show"}
        <p>It's show time!</p>
    {/tmpl_if}
</div>
